packages/plugin-xml/src/DefaultXmlDomParser.ts

```diff
@@ -14,7 +14,10 @@ export class DefaultXmlDomParser extends AbstractParser<Node> {
         // but we don't break it either.
         const element = new VElement({ htmlTag: nodeName(item) });
         if (item instanceof Element) {
-            element.attributes = this.engine.parseAttributes(item);
+            const attributes = this.engine.parseAttributes(item);
+            if (attributes) {
+                element.modifiers.append(attributes);
```

Comment:
Honestly I would just have made it so that `parseAttributes` always return an instance of `Attributes`, even though it is sometimes empty.

Reply:
Well it _is_ actually the case but I check anyway just for fun, because I'm an idiot. :derelict_house: More seriously, I just forgot to change it in this particular instance.